scripts/statesync.bash

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+# microtick and bitcanna contributed significantly here.
+set -uxe
+
+# set environment variables
+export GOPATH=~/go
+export PATH=$PATH:~/go/bin
+
+
+# Install chain-maind
+go install -tags rocksdb ./...
```

Comment:
One need to install the rocksdb c library by himself in advance, right?

Reply:
Do we also need the ldflat: `-X github.com/cosmos/cosmos-sdk/types.DBBackend=rocksdb`?